sktime/transformations/panel/catch22.py

```diff
@@ -14,62 +14,69 @@
-    "SB_TransitionMatrix_3ac_sumdiagcov": _SB_TransitionMatrix_3ac_sumdiagcov,
-    "PD_PeriodicityWang_th0_01": _PD_PeriodicityWang_th0_01,
-}
-FEATURE_NAMES = list(METHODS_DICT.keys())
+
+def get_methods_dict(which="catch22"):
+    from sktime.transformations.panel._catch22_numba import (
+        _catch24_mean,
+        _catch24_std,
+        _CO_Embed2_Dist_tau_d_expfit_meandiff,
+        _CO_f1ecac,
+        _CO_FirstMin_ac,
+        _CO_HistogramAMI_even_2_5,
+        _CO_trev_1_num,
+        _DN_HistogramMode_5,
+        _DN_HistogramMode_10,
+        _DN_OutlierInclude_n_001_mdrmd,
+        _DN_OutlierInclude_p_001_mdrmd,
+        _FC_LocalSimple_mean1_tauresrat,
+        _FC_LocalSimple_mean3_stderr,
+        _IN_AutoMutualInfoStats_40_gaussian_fmmi,
+        _MD_hrv_classic_pnn40,
+        _PD_PeriodicityWang_th0_01,
+        _SB_BinaryStats_diff_longstretch0,
+        _SB_BinaryStats_mean_longstretch1,
+        _SB_MotifThree_quantile_hh,
+        _SB_TransitionMatrix_3ac_sumdiagcov,
+        _SC_FluctAnal_2_dfa_50_1_2_logi_prop_r1,
+        _SC_FluctAnal_2_rsrangefit_50_1_logi,
+        _SP_Summaries_welch_rect_area_5_1,
+        _SP_Summaries_welch_rect_centroid,
+    )
+
+    CATCH22_METHODS_DICT = {
+        "DN_HistogramMode_5": _DN_HistogramMode_5,
+        "DN_HistogramMode_10": _DN_HistogramMode_10,
+        "SB_BinaryStats_diff_longstretch0": _SB_BinaryStats_diff_longstretch0,
+        "DN_OutlierInclude_p_001_mdrmd": _DN_OutlierInclude_p_001_mdrmd,
+        "DN_OutlierInclude_n_001_mdrmd": _DN_OutlierInclude_n_001_mdrmd,
+        "CO_f1ecac": _CO_f1ecac,
+        "CO_FirstMin_ac": _CO_FirstMin_ac,
+        "SP_Summaries_welch_rect_area_5_1": _SP_Summaries_welch_rect_area_5_1,
+        "SP_Summaries_welch_rect_centroid": _SP_Summaries_welch_rect_centroid,
+        "FC_LocalSimple_mean3_stderr": _FC_LocalSimple_mean3_stderr,
+        "CO_trev_1_num": _CO_trev_1_num,
+        "CO_HistogramAMI_even_2_5": _CO_HistogramAMI_even_2_5,
+        "IN_AutoMutualInfoStats_40_gaussian_fmmi": _IN_AutoMutualInfoStats_40_gaussian_fmmi,  # noqa: E501
+        "MD_hrv_classic_pnn40": _MD_hrv_classic_pnn40,
+        "SB_BinaryStats_mean_longstretch1": _SB_BinaryStats_mean_longstretch1,
+        "SB_MotifThree_quantile_hh": _SB_MotifThree_quantile_hh,
+        "FC_LocalSimple_mean1_tauresrat": _FC_LocalSimple_mean1_tauresrat,
+        "CO_Embed2_Dist_tau_d_expfit_meandiff": _CO_Embed2_Dist_tau_d_expfit_meandiff,
+        "SC_FluctAnal_2_dfa_50_1_2_logi_prop_r1": _SC_FluctAnal_2_dfa_50_1_2_logi_prop_r1,  # noqa: E501
+        "SC_FluctAnal_2_rsrangefit_50_1_logi_prop_r1": _SC_FluctAnal_2_rsrangefit_50_1_logi,  # noqa: E501
+        "SB_TransitionMatrix_3ac_sumdiagcov": _SB_TransitionMatrix_3ac_sumdiagcov,
+        "PD_PeriodicityWang_th0_01": _PD_PeriodicityWang_th0_01,
+    }
+    CATCH24_METHODS_DICT = {"DN_Mean": _catch24_mean, "DN_Spread_Std": _catch24_std}
+
+    if which == "catch22":
+        return CATCH22_METHODS_DICT
+    else:
```

Comment:
As you use `"catch24"` in the line 291 maybe it could be forced that the argument here should be `catch24` as well?